packages/transformer/src/IModelImporter.ts

```diff
@@ -257,7 +258,14 @@ export class IModelImporter {
       ) {
         this.onUpdateElement(elementProps);
       } else {
-        this.onInsertElement(elementProps);
+        const doesElementExistInTarget = this.targetDb.elements.tryGetElement(
+          elementProps.id
+        );
+        if (doesElementExistInTarget) {
+          this.onUpdateElement(elementProps);
```

Comment:
I think we should also catch a potential error thrown by onupdateElement and do the duplicateCodeValueMap logic here

Reply:
added a function to catch this potential errors and do the duplicateCodeValueMap logic so that we can add to both places and avoid duplication